internal/database/sqlcommon/contractlisteners_sql.go

```diff
@@ -42,6 +42,7 @@ var (
 		"signature",
 		"topic",
 		"options",
+		"state",
 		"created",
 	}
```

Comment:
I expected to see an "updated" field here (it's in the database migration and in the query factory).

Reply:
This is removed now